README.md

```diff
@@ -1,9 +1,11 @@
 # Chocolate Exploration
+authors: Manvir Kohli, Julie Song, Kelvin Wong
```

Comment:
Frankly we don't really need it for now, since we already have a copyright stating the same information.

But if you would like to keep it, it would be great to use `Authors` instead of `authors`

Reply:
I can remove it no problem..